apps/newsletters-ui/src/app/components/NewslettersTable.tsx

```diff
@@ -3,14 +3,14 @@ import { Link } from 'react-router-dom';
 import type { Column } from 'react-table';
 import type { Newsletter } from '@newsletters-nx/newsletters-data-client';
 import { formatCellBoolean } from './Cell';
-import { Table } from './Table';
+import { Table } from './Table/Table';
```

Comment:
This is just a stylistic thing so not important, but if you have a component defined in a folder, you can call the main component file `index.tsx` so you can import it from './Table' rather than './Table/Table'. It works the same way, but it can help keep the component imports statements on the same level.